solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreaker.java

```diff
@@ -62,6 +64,16 @@ public CircuitBreaker() {}
   /** Get error message when the circuit breaker triggers */
   public abstract String getErrorMessage();
 
+  /**
+   * Get http error code, defaults to {@link SolrException.ErrorCode#TOO_MANY_REQUESTS} but can be
```

Comment:
CI flags this, not sure what the solution is, maybe fully qualified?
```suggestion
   * Get http error code, defaults to {@link org.apache.solr.common.SolrException.ErrorCode#TOO_MANY_REQUESTS} but can be
```

Reply:
No error in IDE, but I re-phrased it without a link ..